test/data/resource-position.json

```diff
@@ -0,0 +1,163 @@
+{
+  "vessels": {
+    "urn:mrn:signalk:uuid:c0d79334-4e25-4245-8892-54e8ccc8021d": {
+      "uuid": "urn:mrn:signalk:uuid:c0d79334-4e25-4245-8892-54e8ccc8021d",
+      "navigation": {
+        "position": {
+          "timestamp": "2015-03-06T16:57:53.643+13:00",
+          "source": {
+            "label": "position",
+            "type": "NMEA0183",
+            "talker": "GP",
+            "sentence": "RMC"
+          },
+          "longitude": 173.1693,
+          "latitude": -41.156426,
+          "altitude": 0
+        },
+        "anchor": {
+          "position":{
+            "timestamp": "2015-03-06T16:57:53.643+13:00",
+            "source": {
+              "label": "position",
+              "type": "NMEA0183",
+              "talker": "GP",
+              "sentence": "RMC"
+            },
+            "longitude": 173.1693,
+            "latitude": -41.156426,
+            "altitude": 0
+          },
+          "currentRadius": {
+            "timestamp": "2015-03-06T16:57:53.643+13:00",
+            "source": {
+              "label": "anchor",
+              "type": "self"
+            },
+            "value":10
+          },
+          "currentRadius": {
+            "timestamp": "2015-03-06T16:57:53.643+13:00",
+            "source": {
+              "label": "anchor",
+              "type": "self"
+            },
+            "value":50
+          },
+          "timestamp": "2015-03-06T16:57:53.643+13:00",
+          "source": {
+            "label": "anchor",
+            "type": "self"
+          }
+        }
+      }
+    }
+  },
+
+  "resources": {
+    "regions":{
```

Comment:
GeoJSON is only a partial solution. It doesnt have timestamps, source, or links to a common waypoint or point list. That makes it a bit of an orphan in signalk, although it could possibly be made to fit. It would be good to reuse the GeoJSON/TopoJSON  stuff though, even if its just for the libraries.